vars/parseStageInfo.groovy

```diff
@@ -112,5 +112,29 @@ def call(Map config = [:]) {
     if (config['ftest_arg']) {
       result['ftest_arg'] = config['ftest_arg']
     }
+
+    result['junit_files'] = 'test_results/*.xml'
+    if (config['junit_files']) {
+      result['junit_files'] = config['junit_files']
+    }
+
+    result['artifacts'] = ['run_test.sh/*', 'vm_test/**']
+    if (config['artifacts']) {
+      result['artifacts'] = config['artifacts']
+    }
+
+    result['valgrind'] = 'disabled'
```

Comment:
Would not be disabled just be the absence of a 'valgrind' key?

Reply:
I'm using the 'disabled' keyword on run_test.sh here https://github.com/daos-stack/daos/pull/3023